python/clim_recal/cli.py

```diff
@@ -65,10 +65,10 @@ def pipeline(
     all_runs: Annotated[bool, typer.Option("--all-runs")] = False,
     default_runs: Annotated[bool, typer.Option("--default-runs")] = False,
     all_methods: Annotated[bool, typer.Option("--all-methods")] = False,
-    cpm_projection: Annotated[bool, typer.Option("--project-cpm")] = True,
-    hads_projection: Annotated[bool, typer.Option("--project-hads")] = True,
-    crop_hads: Annotated[bool, typer.Option("--crop-cpm")] = True,
-    crop_cpm: Annotated[bool, typer.Option("--crop-hads")] = True,
+    cpm_projection: Annotated[bool, typer.Option(" /--skip-cpm")] = True,
```

Comment:
Notes during review discussion:
- default is `True` (= `True), and providing `--skip-cpm` switches to `False` and skips the projection step
```suggestion
    cpm_projection: Annotated[bool, typer.Option(" /--skip-project-cpm")] = True,
```

Reply:
With this version the logic is inverted:
```suggestion
    skip_cpm_projection: Annotated[bool, typer.Option("--skip-project-cpm")] = False,
```